go/oasis-node/cmd/common/flags/flags.go

```diff
@@ -98,7 +104,8 @@ func init() {
 	DebugTestEntityFlags.Bool(CfgDebugTestEntity, false, "use the test entity (UNSAFE)")
 	_ = DebugTestEntityFlags.MarkHidden(CfgDebugTestEntity)
 
-	EntityFlags.StringP(cfgEntity, "e", "", "Path to directory containing entity private key and descriptor")
+	EntityFlags.StringP(cfgEntity, "e", "", "The source of the entity, based on its backing. For file backing, this is the path to the directory containing the entity private key and descriptor. For ledger backing, this is the expected address stored on the device.")
```

Comment:
A bit of a long description. I was wondering if I should split this argument? Or if I just need a more succinct description.